Orange/widgets/utils/itemmodels.py

```diff
@@ -265,15 +265,21 @@ def __delitem__(self, i):
         self.endRemoveRows()
 
     def __setitem__(self, i, value):
+        self._check_sort_order()
         if isinstance(i, slice):
+            self.layoutAboutToBeChanged.emit()
```

Comment:
Is there any reason for layoutAboutToBeChanged/layoutChanged surrounding the begin/endRemoveRows? They are unnecessary (layoutChanged notifies when internal positions of items has changed. e.g. ordering changes, if it is used it should explicitly update persistent indices `self.changePersistentIndex`).

I am not even sure it is supported by views that the model would change dimensions (rowCount/columnCount) between layoutAboutToBeChanged and layoutChanged.